pkg/dataload/constants.go

```diff
@@ -16,6 +16,8 @@ limitations under the License.
 package dataload
 
 const (
+	DATALOAD_LOCK_NAME = "DataLoad"
+
```

Comment:
Naming style for constants: Go follows the camel case naming convention for constants, so I suggest renaming DATALOAD_LOCK_NAME to DataLoadLockName.

Reply:
fixed.